### PR TITLE
Added FATES files

### PR DESCRIPTION
**Pull request by EliasMassoud1**

### CODE REVIEW

File: maat/FATES_Lin2015/init_MAAT_LinInputs_FATES.R [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+#   g0 = dat[c(1:2872),"g0"],
+#   g1_medlyn = dat[c(1:2872),"g1_medlyn"]
+# )
+pars.var    <- NULL
+pars_proc.var <- NULL
+pars_eval.var <- NULL
+
+# define variable ENVIRONMENTAL VARIABLES
+env.var       <- list(
+  ca_conc = dat[c(1:2872),"CO2S"],
```
Comment (nocollier) [inline maat/FATES_Lin2015/init_MAAT_LinInputs_FATES.R]: We shouldn't hard code the number of rows to take. Let's just use them all.

File: maat/FATES_Lin2015/plot_MAAT_FATES_results.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# This script plots the results from the MAAT toolbox
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+# *** READ IN DATA FROM MAAT SIMULATION [MAKE SURE FOLDER PATH IS ACCURATE] ***
+# df = pd.read_csv("/Users/emy/Desktop/Tests/MAAT/_testing_FATES/results/2023-05-30/out_FATES.csv" , usecols=['leaf.ca_conc','A'])
+df = pd.read_csv("/Users/emy/Desktop/Tests/MAAT/_testing_FATES/results/2023-05-30/out_LinInputs_FATES.csv" , usecols=['leaf.ca_conc','leaf.temp','A'], na_values='          NA')
```
Comment (nocollier) [inline maat/FATES_Lin2015/plot_MAAT_FATES_results.py]: We fixed the file paths in the CLM5 version, update here and below.

**Review by nocollier — CHANGES_REQUESTED** [review]

File: maat/FATES_Lin2015/init_MAAT_LinInputs_FATES.R (hunk 1 above)
Comment (EliasMassoud1) [reply maat/FATES_Lin2015/init_MAAT_LinInputs_FATES.R]: This has been corrected now (see here: 4f27243cae02e82cdccb0b8a9a7effe0dea65843)

File: maat/FATES_Lin2015/plot_MAAT_FATES_results.py (hunk 2 above)
Comment (EliasMassoud1) [reply maat/FATES_Lin2015/plot_MAAT_FATES_results.py]: This has been corrected now (see here: https://github.com/rubisco-sfa/MLEcohydrology/commit/4f27243cae02e82cdccb0b8a9a7effe0dea65843)

**Comment by nocollier** [comment]
This looks good, you checked that it runs on your machine?

**Review by nocollier — APPROVED** [review]

**Pull request merged**